spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/source/TestSparkTable.java

```diff
@@ -53,4 +54,68 @@ public void testTableEquality() throws NoSuchTableException {
     assertThat(table1).as("References must be different").isNotSameAs(table2);
     assertThat(table1).as("Tables must be equivalent").isEqualTo(table2);
   }
+
+  @TestTemplate
+  public void testTimeTravelEquality() throws NoSuchTableException {
+    CatalogManager catalogManager = spark.sessionState().catalogManager();
+    TableCatalog catalog = (TableCatalog) catalogManager.catalog(catalogName);
+    Identifier identifier = Identifier.of(tableIdent.namespace().levels(), tableIdent.name());
+
+    sql("INSERT INTO TABLE %s VALUES (1, 'a')", tableName);
+
+    SparkTable table1 = (SparkTable) catalog.loadTable(identifier);
+    final long version1Snapshot = table1.table().currentSnapshot().snapshotId();
```

Comment:
Same as in other places, no final keywords for local vars.

Reply:
Removed. Thanks.